src/inspectors/CollaborationInspector.spec.ts

```diff
@@ -56,4 +61,30 @@ describe('Collaboration Inspector', () => {
     const response = await inspector.getPullCommits('octocat', 'Hello-World', 1);
     expect(response).toMatchObject(getPullCommitsServiceResponse);
   });
+
+  it('returns max number of pull requests', async () => {
+    bitbucketNock = new BitbucketNock('pypy', 'pypy');
+    containerCtx.container.rebind(Types.IContentRepositoryBrowser).to(BitbucketService);
+    const collaborationInspector = containerCtx.container.get<CollaborationInspector>(Types.ICollaborationInspector);
+
+    bitbucketNock.getOwnerId();
+    bitbucketNock.getApiResponse({
+      resource: 'pullrequests',
+      state: BitbucketPullRequestState.closed,
+      pagination: { page: 1, perPage: 5 },
+    });
+
+    const response = await collaborationInspector.getPullRequests('pypy', 'pypy', {
+      maxNumberOfPullRequests: 1,
+      filter: { state: PullRequestState.closed },
+      pagination: { page: 1, perPage: 5 },
+    });
+
+    const getMaxOneClosedPullRequestsResponse = bitbucketNock.mockBitbucketPullRequestsResponse({
```

Comment:
note: Quite long var name. It'd be better to have it directly in `.toEqual(...)`. It seems that it does something.